--- a/npm/private/npm_translate_lock_generate.bzl
+++ b/npm/private/npm_translate_lock_generate.bzl
@@ -101,14 +101,5 @@ def generate_repository_files(rctx, pnpm_lock_label, importers, packages, patch
         "BUILD.bazel": [
             """load("@bazel_skylib//:bzl_library.bzl", "bzl_library")""",
             "",
-            """
-# A no-op run target that can be run to invalidate the repository
-# to update the pnpm lockfile. Useful under bzlmod where
-# `bazel sync --only=repo` is a no-op.
-sh_binary(
-    name = "sync",
-    srcs = ["@aspect_rules_js//npm/private:noop.sh"],
-)""",
-            "",
             "exports_files({})".format(starlark_codegen_utils.to_list_attr([
                 rctx.attr.defs_bzl_filename,
                 rctx.attr.repositories_bzl_filename,
